test/hotspot/jtreg/runtime/cds/appcds/jcmd/JCmdTestFileSafety.java

```diff
@@ -41,8 +41,25 @@
 import jdk.test.lib.cds.CDSTestUtils;
 import jdk.test.lib.apps.LingeredApp;
 import jdk.test.lib.Platform;
+import jdk.test.lib.process.OutputAnalyzer;
 
 public class JCmdTestFileSafety extends JCmdTestDumpBase {
+    static final String promtStdout = "please check stdout file";
+    static final String promtStderr = "or stderr file";
+
+    static void checkContainAbsoluteLogPath(OutputAnalyzer output) throws Exception {
+       String stdText = output.getOutput();
+       if (stdText.contains(promtStdout) &&
+           stdText.contains(promtStderr)) {
```

Comment:
What if either `promtStdout` or `promtStderr` is not found in stdout, should a RuntimeException be thrown as well?

Reply:
The exception could be other type and without containing those two strings. It should leave to the default processing. Only with those two messaged found, we check if the file with a absolute path.